axelar-chains-config/info/testnet.json

```diff
@@ -2517,6 +2722,21 @@
           "codeId": 26,
           "address": "axelar1ce9rcvw8htpwukc048z9kqmyk5zz52d5a7zqn9xlq2pg0mxul9mqxlx2cq"
         },
+        "sui": {
+          "governanceAddress": "axelar10d07y265gmmuvt4z0w9aw880jnsr700j7v9daj",
+          "serviceName": "amplifier",
+          "sourceGatewayAddress": "0x6ddfcdd14a1019d13485a724db892fa0defe580f19c991eaabd690140abb21e4",
+          "votingThreshold": [
+            "51",
+            "100"
+          ],
+          "blockExpiry": 10,
+          "confirmationHeight": 100000,
```

Comment:
Even if a value is being ignored, we should update this param on-chain to avoid confusion (check if voting verifier has the ability to update yet, if not ticket this change, and ask core team to add this feature in the future)